packages/core/src/index.ts

```diff
@@ -203,12 +210,37 @@ export const ListRef = InternalListRef as new <Types extends SchemaTypes, T, P =
   nullable: boolean,
 ) => PothosSchemaTypes.ListRef<Types, T, P>;
 
+export type FieldRef<
+  Types extends SchemaTypes,
+  T = unknown,
+  Kind extends FieldKind = FieldKind,
+> = PothosSchemaTypes.FieldRef<Types, T, Kind>;
+export const FieldRef = InternalFieldRef as new <
+  Types extends SchemaTypes,
+  T = unknown,
+  Kind extends FieldKind = FieldKind,
+>(
+  kind: Kind,
+  initConfig: (name: string, typeConfig: PothosTypeConfig) => PothosOutputFieldConfig<Types>,
+) => PothosSchemaTypes.FieldRef<Types, T, Kind>;
+
+export type InputFieldRef<Types extends SchemaTypes, T> = PothosSchemaTypes.InputFieldRef<Types, T>;
+export const InputFieldRef = InternalInputFieldRef as new <Types extends SchemaTypes, T>(
+  initConfig: (name: string, typeConfig: PothosTypeConfig) => PothosInputFieldConfig<Types>,
+) => PothosSchemaTypes.InputFieldRef<Types, T>;
+
+export type ArgumentRef<Types extends SchemaTypes, T> = PothosSchemaTypes.ArgumentRef<Types, T>;
+export const ArgumentRef = InternalArgumentRef as new <Types extends SchemaTypes, T>(
```

Comment:
This just makes it so these types of Refs can be more easily extended by plugins